README.md

```diff
@@ -102,6 +103,22 @@ docker-compose -f docker-compose-embedding.yml run build
 
 Assets are saved to `ui/admin/dist/`.
 
+### Building Desktop UI for Production
+
+See ui/desktop/README.md Building for Production for more information on how to
+build the Desktop UI.  Be sure to set build-related environment variables as
+necessary for your target environment, as described in the Desktop UI readme.
+
+To quickly produce a production build of Desktop UI, run from this folder:
+
+```bash
+yarn install
+yarn build:ui:desktop
+```
+
+Assets are saved to `ui/desktop/electron-app/out/make/`.
+DMG packaged desktop UI is available at asset location as `Boundary.dmg`.
```

Comment:
Should/can we include the version of the DT client in this filename?

Reply:
Good idea. Will work on this next.